### PR TITLE
[feat] Add typescript definition file

### LINKED ISSUE #153: Typescript types for sketch-module-web-view
- was curious if this repo has an npm package for typescript types? Similar to @sketch-hq/sketch-file-format-ts, but maybe @sketch-hq/sketch-module-web-view-ts?

Thank You!

### PR DESCRIPTION
**Pull request by lizheming**
close #153 after pull request merged

### CODE REVIEW

**Comment by lizheming** [comment]
hello @mathieudutour, do you have time to review the pr?

File: package.json [hunk 1]
```diff
@@ -15,7 +16,8 @@
     "eslint-plugin-prettier": "^3.0.1",
     "lint-staged": "^8.1.5",
     "pre-commit": "^1.2.2",
-    "prettier": "^1.17.0"
+    "prettier": "^1.17.0",
+    "sketch-app-types": "^0.1.1"
```
Comment (mathieudutour) [inline package.json:20]: if you reference it from the types declaration, it needs to be in `dependencies`

But is it needed? I had a quick look at the types and couldn't find a reference to it?

Comment (lizheming) [reply package.json:20]: `sketch-app-types` is a basic type definition module for `sketch` object. `BrowserWindowOptions` has `parent` property which type is sketch Document. the Document definition depends on `sketch/dom`, so I add `sketch-app-types` dependency for our module.
I think the definition module just need in development environment, so I add it to devdependencies. It's not a big problem, I'll change it~

**Comment by lizheming** [comment]
Hello @mathieudutour, I have modified by your advice, please review again. By the way, I also add ts support for other three module https://github.com/skpm/dialog/pull/13 https://github.com/skpm/fs/pull/31 https://github.com/skpm/path/pull/3 , please take a little time to check them. Thanks a lot!

**Comment by lizheming** [comment]
Hey @mathieudutour I have removed `sketch-app-types` dependency to avoid your doubt, please check it again, thank you~

**Review by mathieudutour — APPROVED** [review]
Thanks

**Pull request merged**